### DISCUSSION TITLE
Hide column by default?

### DISCUSSION THREAD

**Discussion by MACscr** [rappasoft/laravel-livewire-tables#366] [Q&A] [answered]
Is there an option to hide a column by default? I dont want the ID column seen by default, but i do want it to be in the user selection drop down option.

**Comment by rappasoft** [comment]
There is currently no way to do this, I couldn't find a good way to have it work well with what was in the session vs. excluded so I left it out for now. I will probably make the whole feature over at some point and when I do I'll make sure to remember to factor it in.

**Reply by LaMance22** [reply]
Hi, thanks for this package. Have you solved the default of hiding columns?

**Comment by aphoe** [comment, marked as answer]
Try this... 

`Column::make('Name')->hideIf(true)`

**Reply by iamgoodbytes** [reply]
Cool that works!
I wanted to load the $id because I need it to make the row clickable, but didn't want to show it

**Reply by lrljoe** [reply]
You can use setAdditionalSelects for extra fields to be available.
This saves a chunk of processing.
Only use hideIf for conditional display of a column
